packages/components/src/components/Table/TableCell.tsx

```diff
@@ -1,10 +1,12 @@
-import { spacingsPx, typography } from '@trezor/theme';
 import { ReactNode } from 'react';
 import styled, { css } from 'styled-components';
 
+import { spacingsPx, typography } from '@trezor/theme';
+
 export interface TableCellProps {
-    children: ReactNode;
+    children?: ReactNode;
     $alignRight?: boolean;
+    $withRightMargin?: boolean;
```

Comment:
1) Btw since it's in the DS it should have predictable API so we shouldn't use $ which means we need to create a wrapper above styled component and use `makeTransientProps`

2) Also thinking that maybe using `margin` from `frameProps` would be more universal than `withRightMargin`

3) please could you update API to `align = 'left' | 'right'`? 🙏

Reply:
Yes make sense 👍